### PR TITLE
refactor error messages with project id when `MiniAppNotFoundException` exception

### PR DESCRIPTION
**Pull request by khairul-alam-licon**
# Description
Refactor messages with specifying project id when `MiniAppNotFoundException` exception. 

## Links
MINI-2175

# Checklist
- [x] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

**Review by corycaywood — APPROVED** [review]
👍

File: miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/api/ApiClient.kt [hunk 1]
```diff
@@ -111,7 +111,7 @@ internal class RetrofitRequestExecutor(
         }
     }
 
-    @Throws(MiniAppSdkException::class)
+    @Throws(MiniAppSdkException::class, MiniAppNotFoundException::class)
```
Comment (minh-rakuten) [inline miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/api/ApiClient.kt:114]: `MiniAppNotFoundException` is `MiniAppSdkException` so I don't think we need to add it here.

Comment (khairul-alam-licon) [reply miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/api/ApiClient.kt:114]: I have followed the way like `MiniApp.create` where are keeping both. I think, the main reason is distinguish both as separate exception while providing error message.

**Review by minh-rakuten — APPROVED** [review]
👍

**Review by corycaywood — APPROVED** [review]

**Pull request merged**